build.gradle

```diff
@@ -13,11 +26,12 @@ plugins {
 }
 
 apply plugin: 'net.minecraftforge.gradle.forge'
+
 //Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 
-version = "1.5.6"
-group = "com.blakebr0.extendedcrafting"
-archivesBaseName = "ExtendedCrafting-1.12.2"
+group = "omnifactorydevs.morefurnaces" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
```

Comment:
I believe this should say `omnifactorydevs.extendedcrafting`.

Reply:
😩